### ISSUE TITLE
Is there any hope running sioyek with OpenGL 2.0?

### ISSUE THREAD

**Issue by ephemer4l** [ahrm/sioyek#1184]
I want to run sioyek on my Thinkpad x200, which has GM45, which only supports OpenGL 2.0 at most. When trying to launch sioyek, I am getting a lot of errors in the form
```
QEGLPlatformContext: Failed to create context: 3009
QRhiGles2: Failed to create temporary context
```
which I assume are there because I don't have OpenGL 3.2. Can we run with 2.0?

**Comment by ahrm** [comment]
Yes, the next version of sioyek will have ability to run without opengl at all.

**Comment by ephemer4l** [comment]
To be clear, this is not yet possible, even with the git version, right?

**Comment by ahrm** [comment]
> To be clear, this is not yet possible, even with the git version, right?

right.

**Comment by hamdan7m** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by ephemer4l** [comment]
@ahrm Could you tell me what this compile-time option is?

**Comment by ahrm** [comment]
It is not available in the public version yet. It will hopefully be released in the next few months.
